config/azure/build.yaml

```diff
@@ -80,24 +87,35 @@ jobs:
    - script: |
        echo BUILD_TYPE=${{ parameters.buildType }}
        g++ --version
-       scons -j ${{ parameters.threads }} package ENV_VARS_TO_IMPORT=PATH BUILD_TYPE=${{ parameters.buildType }} DELIGHT_ROOT=$DELIGHT ARNOLD_ROOT=$ARNOLD_ROOT BUILD_DIR=./build INSTALL_DIR=./install/$(Gaffer.Build.Name) BUILD_CACHEDIR=sconsCache
+       rm -rf .sconsign.dblite .sconf_temp
+       scons -j ${{ parameters.threads }} build ENV_VARS_TO_IMPORT=PATH BUILD_TYPE=${{ parameters.buildType }} DELIGHT_ROOT=$DELIGHT ARNOLD_ROOT=$ARNOLD_ROOT BUILD_DIR=./build INSTALL_DIR=./install/$(Gaffer.Build.Name) BUILD_CACHEDIR=sconsCache SPHINX=$(Gaffer.Sphinx)
      displayName: 'Build'
      env:
        DISPLAY: :99.0
        AZURE: 1
 
    - script: |
-       ./install/$(Gaffer.Build.Name)/bin/gaffer test -stopOnFailure
+       ./build/bin/gaffer test -stopOnFailure
      displayName: 'Test'
      env:
        DISPLAY: :99.0
        AZURE: 1
        USER: azureTestUser # ImageWriterTest requires $USER but by default Azure doesn't provide it
        ARNOLD_LICENSE_ORDER: none # Don't waste time looking for a license that doesn't exist
 
+   - script: |
+       echo BUILD_TYPE=${{ parameters.buildType }}
+       g++ --version
```

Comment:
Do we need to repeat this? We shouldn't even be invoking the compiler in this build step, so it seems a little unnecessary.

Reply:
removed